iamy/aws.go

```diff
@@ -176,6 +190,27 @@ func (a *AwsFetcher) marshalRoleDescriptionAsync(roleName string, target *string
 	}()
 }
 
+func (a *AwsFetcher) populateInstanceProfileData(resp *iam.ListInstanceProfilesOutput) error {
+	for _, profileResp := range resp.InstanceProfiles {
+		if cfnResourceRegexp.MatchString(*profileResp.InstanceProfileName) {
+			log.Printf("Skipping CloudFormation generated instance profile %s", *profileResp.InstanceProfileName)
+			continue
+		}
+		profile := InstanceProfile{iamService: iamService{
+			Name: *profileResp.InstanceProfileName,
+			Path: *profileResp.Path,
+		}}
+		if len(profileResp.Roles) > 0 {
+			for _, roleResp := range  profileResp.Roles {
```

Comment:
double spaces after `range`?

Reply:
Yep probably a good idea to run a `go fmt` over these modified files. I typically add a hook to my editor to run `goimports` on save @andrewjhumphrey

Reply:
043328d